Taxes/Integration/01_ManageTaxRates.md

```diff
@@ -0,0 +1,201 @@
+
+1. Click the tax rate to be edited in the list of tax rates.  
+The *Edit tax rate* view is displayed.
+
+  ![Edit tax rate](../../Assets/Screenshots/Taxes/Settings/TaxRates/EditTaxRate.png "[Edit tax rate]")
+
+2. Edit the desired data in the corresponding field(s).
+
+  > [Info] Be aware that any changes made to an existing tax rate will overwrite the previously configured values.
+
+3. Click the [SAVE] button.  
+The *Edit tax rate* view is closed and the list of tax rates is displayed again.  
+
+4. Click the ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh) button in the upper right corner.   
+The edited tax rate is displayed in the list of tax rates.
+
+
+## Delete a tax rate
+
+Delete a tax rate that is no longer applicable.
+
+> [Warning] Be aware that any deletion is permanent and cannot be undone.
+
+#### Prerequisites
+
+A tax rate has been created, see [Create a tax rate](#create-a-tax-rate).
+
+#### Procedures
+
+*Taxes > Settings > Tab TAX RATES*
+
+![Tax rates](../../Assets/Screenshots/Taxes/Settings/TaxRates/TaxRates.png "[Tax rates]")
+
+1. Select the checkbox of the tax rate to be deleted.  
+The editing toolbar is displayed.
+
+  > [Warning] Be aware that the existing tax rate will be deleted permanently.
+
+2. Click the [DELETE] button in the upper right corner.  
+The selected tax rate has been deleted.
+
+3. Click the ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh) button in the upper right corner.   
+The deleted tax rate is removed from the list of tax rates.
+
+[comment]: <> (17.08.22 - DELETE Button not working. Bug reported. Check again when fixed if Refresh button is needed here.)
+
+
+## Define the tax rates for delivery thresholds
+
+Define the necessary tax rates to include tax delivery thresholds in the system configuration.
+
+> [Info] As of July 2021, a new EU-wide threshold of 10,000 EUR applies. As soon as this threshold has been reached, all products must be taxed in the ship-to (delivery) country to the corresponding delivery country tax rate. For detailed information about the new cross-border taxation rules, see [VAT - One Stop Shop](https://vat-one-stop-shop.ec.europa.eu/index_en) or consult a tax advisor.
+
+#### Prerequisites
+
+- All applicable tax classes have been created, see [Create a tax class](./02_ManageTaxClasses.md#create-a-tax-class).
+- All applicable tax zones have been created, see [Create a tax zone](./03_ManageTaxZones.md#create-a-tax-zone).
+
+#### Procedures
+
+*Taxes > Settings > Tab TAX RATES*
+
+![Tax rates](../../Assets/Screenshots/Taxes/Settings/TaxRates/TaxRates.png "[Tax rates]")
+
+[comment]: <> (Unsicher - Subprocedures mit H3? Beide gehören zusammen, denn man braucht zwei Tax rates mit zwei unteschiedlichen Gültigkeitszeiträume. Problem mit selben Tax zone wäre dann für die Decision Matrix, denn da kann man bei der Einstellung von jeder Zeile unterscheiden können.)
```

Comment:
Ja, kannst du beide mit H3 machen, dann aber jeweils mit h4 prerequisites und procedure für jede der beiden Prozeduren. 
Den zweiten Teil musst du mir nochmal erklären, wenn ich wieder da bin, das verstehe ich nicht mit der tax zone in der decision matrix =D